### DISCUSSION TITLE
Program crash

### DISCUSSION THREAD

**Discussion by dylu6699** [libuv/libuv#3276] [Q&A]
Thread 1 "sdp" received signal SIGSEGV, Segmentation fault.
0x0000000000000000 in ?? ()
(gdb) bt
#0  0x0000000000000000 in ?? ()
#1  0x00000000004bff1d in uv__read (stream=stream@entry=0x761a70) at /mnt/hgfs/gitee/metel_sdp/client/third_party/libuv1.38.0/src/unix/stream.c:1202
#2  0x00000000004c01c8 in uv__stream_io (loop=<optimized out>, w=0x761af8, events=1)
    at /mnt/hgfs/gitee/metel_sdp/client/third_party/libuv1.38.0/src/unix/stream.c:1306
#3  0x00000000004c53a8 in uv__io_poll (loop=loop@entry=0x74b3a0 <default_loop_struct>, timeout=188)
    at /mnt/hgfs/gitee/metel_sdp/client/third_party/libuv1.38.0/src/unix/linux-core.c:431
#4  0x00000000004ba8a5 in uv_run (loop=0x74b3a0 <default_loop_struct>, mode=mode@entry=UV_RUN_DEFAULT)
    at /mnt/hgfs/gitee/metel_sdp/client/third_party/libuv1.38.0/src/unix/core.c:381
#5  0x00000000004af850 in uv::EventLoop::run (this=0x74b120 <uv::EventLoop::DefaultLoop()::defaultLoop>)
    at /mnt/hgfs/gitee/metel_sdp/client/third_party/uv-cpp1.5.4/uv/EventLoop.cpp:67
#6  0x0000000000432dfb in main () at /mnt/hgfs/gitee/metel_sdp/client/sdp/app/main.cpp:55

**Comment by santigimeno** [comment]
Can you share a reduced test case that reproduces the issue?
